<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1187</width>
    <height>844</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>900</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WebWatcher</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/webscanner-watch.ico</normaloff>../icons/webscanner-watch.ico</iconset>
  </property>
  <widget class="QWidget" name="mainwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="addSiteButton">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeSiteButton">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="subsView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QFormLayout" name="formLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="siteLabel">
           <property name="text">
            <string>Website</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="siteEdit"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="queryLabel">
           <property name="text">
            <string>JavaScript query</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="queryEdit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="intervalLabel">
           <property name="text">
            <string>Update Interval</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="intervalEdit"/>
           </item>
           <item>
            <widget class="QComboBox" name="intervalUnitsCombobox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="sizeIncrement">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="updateSiteButtion">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="titleLabel">
           <property name="text">
            <string>Title (optional)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="titleEdit"/>
         </item>
        </layout>
       </widget>
       <widget class="QTableView" name="requestHistoryView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>addSiteButton</tabstop>
  <tabstop>removeSiteButton</tabstop>
  <tabstop>subsView</tabstop>
  <tabstop>siteEdit</tabstop>
  <tabstop>titleEdit</tabstop>
  <tabstop>queryEdit</tabstop>
  <tabstop>intervalEdit</tabstop>
  <tabstop>intervalUnitsCombobox</tabstop>
  <tabstop>updateSiteButtion</tabstop>
  <tabstop>requestHistoryView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>addSiteButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>handleAddSiteButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeSiteButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>handleRemoveSiteButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subsView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>handleSubsClick(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subsView</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>handleSubsDoubleClick(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateSiteButtion</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>handleSubsEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>895</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>handleAddSiteButton()</slot>
  <slot>handleRemoveSiteButton()</slot>
  <slot>handleSubsDoubleClick(QModelIndex)</slot>
  <slot>handleSubsClick(QModelIndex)</slot>
  <slot>handleSubsEdit()</slot>
 </slots>
</ui>
